tests/unittests/test_upgrade.py

```diff
@@ -48,5 +53,6 @@ def test_paths_has_run_dir_attribute(self, previous_obj_pkl):
         assert previous_obj_pkl.paths.run_dir is not None
 
     def test_vendordata_exists(self, previous_obj_pkl):
-        assert previous_obj_pkl.vendordata2 is None
-        assert previous_obj_pkl.vendordata2_raw is None
+        if previous_obj_pkl.dsname != "Azure":
```

Comment:
The change above should help prevent a regression, but I don't really understand the change in `test_vendordata_exists()`. Why does skipping when the pickle ds isn't Azure help?